js/tic_tac_toe.js

```diff
@@ -1,2 +1,84 @@
-// Your code here
-// Use the jQuery
+var count = 0;
+var blue_rows = [];
+var red_rows = [];
+var blue_columns = [];
+var red_columns = [];
+var $lFirst, $lSecond, $lThird, $rFirst, $rSecond, $rThird, $heading;
+
+$(document).ready(function() {
+  $cell = $('.box_cell');
+  $lFirst = $('#box_0_0');
```

Comment:
Keeping everything in an array might be cleaner in the long term, but this will work.